dist/formats/answer/frontend/schema.json

```diff
@@ -137,6 +137,10 @@
           "description": "Link type automatically added by Publishing API",
           "$ref": "#/definitions/frontend_links_with_base_path"
         },
+        "secondary_to_step_by_steps": {
```

Comment:
Should this be called `secondary_to_step_navs`, so that it follows the same pattern as `part_of` and `related_to` and matches the reverse link `pages_secondary_to_step_nav`?